packages/server/src/routes/members/index.ts

```diff
@@ -760,23 +778,29 @@ export const verify = (
         return existingVerifyOperations.docs[0].ref;
       }
 
-      await movePrivateVideoToPublicVideo(
-        config,
-        storage,
-        loggedInUid,
-        videoToken
-      );
+      if (!videoUrl) {
+        // To satisfy the type compiler (false should be impossible due to the above missing params check)
```

Comment:
Talked offline, can rework the definition of `VerifyMemberApiCall` to only allow one of `videoToken` or `videoUrl` since that's the intended effect, and then the types will fall into place automatically without a hack like that.